numpy/lib/_npyio_impl.pyi

```diff
@@ -150,19 +150,21 @@ def save(
 def save(
     file: str | os.PathLike[str] | _SupportsWrite[bytes],
     arr: ArrayLike,
-    allow_pickle: bool,
+    allow_pickle: bool = ...,
     fix_imports: bool,
     /,
 ) -> None: ...
 
 def savez(
     file: str | os.PathLike[str] | _SupportsWrite[bytes],
+    allow_pickle: bool = ...,
```

Comment:
`allow_pickle` should come after the `*args`

Reply:
:upside_down_face: Would really help to have a way to check these.

Reply:
Ah, I see pylsp *does* in fact catch that error.